src/catalog/catalog.cpp

```diff
@@ -157,8 +157,9 @@ Result Catalog::CreatePrimaryIndex(const std::string &database_name,
     key_schema->SetIndexedColumns(key_attrs);
 
     index_metadata = new index::IndexMetadata(
-        "customer_pkey", GetNextOid(), INDEX_TYPE_BWTREE,
-        INDEX_CONSTRAINT_TYPE_PRIMARY_KEY, schema, key_schema, key_attrs, true);
+        "customer_pkey", GetNextOid(), table->GetOid(), database->GetOid(),
+        INDEX_TYPE_BWTREE, INDEX_CONSTRAINT_TYPE_PRIMARY_KEY, schema,
+        key_schema, key_attrs, true);
```

Comment:
use local variable "bool unique = true" and use unique as the argument, its more readable.